cmd/kubeadm/app/discovery/token/token.go

```diff
@@ -159,8 +159,7 @@ func RetrieveValidatedClusterInfo(cfg *kubeadmapi.NodeConfiguration) (*clientcmd
 
 		fmt.Printf("[discovery] Cluster info signature and contents are valid and TLS certificate validates against pinned roots, will use API Server %q\n", endpoint)
 		return secureKubeconfig, nil
-	})
-	if err != nil {
```

Comment:
Can you explain why this was shadowed? I may miss something here :)

Reply:
I think the "err" variable was declared twice in line 48 and line 62

Reply:
It didn't pass `gofmt` check, might be my mistake, just change it back, will try to figure it out by myself :)

Reply:
OK :)